internal/service/vpclattice/resource_configuration.go

```diff
@@ -372,9 +372,15 @@ func (r *resourceConfigurationResource) Delete(ctx context.Context, request reso
 
 	conn := r.Meta().VPCLatticeClient(ctx)
 
-	_, err := conn.DeleteResourceConfiguration(ctx, &vpclattice.DeleteResourceConfigurationInput{
-		ResourceConfigurationIdentifier: fwflex.StringFromFramework(ctx, data.ID),
-	})
+	// Handle EventBridge-managed resource association deletion.
+	const (
+		timeout = 1 * time.Minute
+	)
```

Comment:
Observed a test failure on my first run of this suite with the error retried below. Not sure if this warrants bumping the default and/or making it configurable?

```
=== NAME  TestAccEventsConnection_invocationConnectivityParameters
    connection_test.go:596: Error running post-test destroy, there may be dangling resources: exit status 1

        Error: deleting VPCLattice Resource Configuration (rcfg-0ecdf92d7a056f2d1)

        operation error VPC Lattice: DeleteResourceConfiguration, https response
        error StatusCode: 400, RequestID: e4a27088-0707-41c4-a4df-30a71d643c21,
        ValidationException: rcfg-0ecdf92d7a056f2d1 has existing association with
        service networks
```